### PR TITLE
check for self._executor in Execute

### PR DESCRIPTION
**Pull request by wallacbe**
https://github.com/google/openhtf/issues/275

### CODE REVIEW

**Comment by wallacbe** [comment]
output with change:

python examples/hello_world.py 
Provide a DUT ID in order to start the test.
test
Hello World!
^C2016-05-24 15:28:02,440 - ERROR - Received SIGINT.
2016-05-24 15:28:02,440 - ERROR - Stopping Test due to SIGINT
2016-05-24 15:28:02,441 - ERROR - Test state: <TestState: test@wallacbe Running Phase: hello_world>

**Comment by wallacbe** [comment]
PTAL

File: openhtf/__init__.py [hunk 1]
```diff
@@ -185,9 +185,10 @@ def Execute(self, test_start=None, loop=None):
     finally:
       # If the framework doesn't transition from INITIALIZING to EXECUTING
       # then test state isn't set and there's no record to output.
-      if self._executor.GetState():
-        record = self._executor.GetState().GetFinishedRecord()
-        self.OutputTestRecord(record)
+      if self._executor:
```
Comment (grybmadsci) [inline openhtf/__init__.py]: `if self._executor and self._executor.GetState():`

Comment (wallacbe) [inline openhtf/__init__.py]: Done

**Comment by wallacbe** [comment]
PTAL

**Comment by fahhem** [comment]
LGTM

**Pull request merged**